lnst/Tests/XDPBench.py

```diff
@@ -108,7 +108,6 @@ class XDPBench(BaseTestModule):
     interval = IntParam(default=1)
 
     redirect_device = DeviceParam()
-    xdp_mode = ChoiceParam(type=StrParam, choices=("native", "skb"), default="native")
```

Comment:
removing this line should be done in a different commit since it is readded as a different type